### PR TITLE
Ephemeral containers support

### PR DESCRIPTION
**Pull request by rkmathi**
resolves: #105 

~This PR is depends on https://github.com/stern/stern/pull/124.~
~I will rebase this when #124 is resolved.~

Support ephemeral containers logs.

### CODE REVIEW

File: stern/watch.go [hunk 1]
```diff
@@ -107,25 +111,29 @@ func Watch(ctx context.Context, i v1.PodInterface, podFilter *regexp.Regexp, exc
 						}
 					}
 				case watch.Deleted:
-					var containers []corev1.Container
-					containers = append(containers, pod.Spec.Containers...)
+					var containerNames []string
```
Comment (rkmathi) [inline stern/watch.go:114]: Container and InitContainer is [`k8s.io/api/core/v1.Container`](https://github.com/kubernetes/api/blob/v0.21.0/core/v1/types.go#L2201), but EphemeralContainer is
[`k8s.io/api/core/v1.EphemeralContainer`](https://github.com/kubernetes/api/blob/v0.21.0/core/v1/types.go#L3416).
So I changed receiving container name directly.

**Review by superbrothers — APPROVED** [review]
I'm sorry there is no test :bow:

**Comment by superbrothers** [comment]
@rkmathi Thanks for your contribution🙌

**Comment by rkmathi** [comment]
Thank you for your quick review 😸

**Pull request merged**